### ISSUE TITLE
[SUGGESTION] Blazor Web App New Project - Missing Auto render mode project

### ISSUE THREAD

**Issue by jemcmullin** [microsoft/vscode-dotnettools#836]
### Describe the feature you'd like

Using the Command Palette for new project templates, Blazor Web App template option (essentially "dotnet new blazor") could be improved by prompting for a few more options seen in Visual Studio. Currently using the CLI for new projects with the option (-int|--interactivity <None|Server|Webassembly|Auto >).

Other options for this template could be useful such as Include Auth, etc. Referenced:
https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new-sdk-templates#blazor

### Alternatives considered

Add an additional template for Blazor Web App with interactivity Auto. Seems important as it's different solution setup than current options and was highly promoted with .NET8 release.

**Comment by julioct** [comment]
Yes, please, I'm trying to create an empty Blazor project with no interactivity and there's no way to do this with the C# Dev Kit project templates.

Only way to do it is via the CLI, like this:

```bash
dotnet new blazor --interactivity None --empty -n MyProject
```

**Comment by smitpatel** [comment]
This is the list of parameters shown in UI for improved experience
![image](https://github.com/microsoft/vscode-dotnettools/assets/1528107/7dec2b19-d9c8-4671-90ef-370742cfdbc9)

**Comment by smitpatel** [comment]
This is fixed in v1.8.14
